build/compile/Microsoft.DotNet.Cli.LzmaArchive.targets

```diff
@@ -80,12 +80,12 @@
 
       <Message Text="Publishing Archiver" />
 
-      <DotNetPublish ToolPath="$(Stage0Directory)"
+      <DotNetPublish ToolPath="$(Stage2Directory)"
```

Comment:
And everywhere else. Basically, we build our sources with stage0, which have Compile Include. If you use stage2 to build other things that depend on sources, they can't have Compile Include, but sources can't be without it because of stage0.